modules/microshift-updates-rhde-config-rhel-repos.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+//microshift_updating/microshift-update-options.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-updates-rhde-config-rhel-repos_{context}"]
+= Keeping {microshift-short} and {op-system-base} in a supported configuration
+
+When using RPM updates, avoid creating an unsupported configuration or breaking your cluster by carefully managing your {op-system-base} repositories.
+
+.Repository management steps
+
+* Avoid unintended updates by locking your operating system version with the following command:
++
+[source,terminal]
+----
+$ sudo subscription-manager release --set=x.y
+----
+
+* To update both {microshift-short} and {op-system-base} versions, use the following command:
++
+[source,terminal]
+----
+$ sudo subscription-manager release --set=<9.4> command. # <1>
+----
+<1> Replace _<9.4>_ with the major and minor version of your compatible {op-system-base} system.
+
+* If you are using an EUS {microshift-short} release, first disable the {op-system-base} standard-support-scope repositories by using the following command:
++
+[source,terminal]
+----
+$ sudo subscription-manager repos \
+    --disable=rhel-<9>-for-x86_64-appstream-rpms \ # <1>
+    --disable=rhel-<9>-for-x86_64-baseos-rpms
+----
+<1> Replace _<9>_ with the major version of your compatible {op-system-base} system.
++
+* After you disabled the standard-support repositories, enable the {op-system-base} EUS repos using the following command:
++
+[source,terminal]
+----
+$ sudo subscription-manager repos \
+    --enable rhel-<9>-for-x86_64-appstream-eus-rpms \  # <1>
+    --enable rhel-<9>-for-x86_64-baseos-eus-rpms`
+----
+<1> Replace _<9>_ with the major version of your compatible {op-system-base} system.
+
+.  List the repositories you have enabled for {op-system-base} by running the following command:
```

Comment:
This bit isn't rendering correctly in the [preview](https://87660--ocpdocs-pr.netlify.app/microshift/latest/microshift_updating/microshift-update-options.html#microshift-updates-rhde-config-rhel-repos_microshift-update-options).